### PR TITLE
chore: update uta docker image to uta_20210129b

### LINKED ISSUE #253: Update uta docker version
In [this](https://github.com/biocommons/uta/issues/228) issue from biocommons.uta, a new version of UTA has been uploaded. We should update our docker-compose to point to this new version

### PR DESCRIPTION
**Pull request by korikuzma**
close #253

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -17,12 +17,12 @@ services:
...
       - 5000:5000
-  
+
   uta:
     # Test:
-    # psql -XAt postgres://anonymous@localhost/uta -c 'select count(*) from transcript'
-    # 249909
-    image: biocommons/uta:uta_20180821
+    # psql -XAt postgres://anonymous@localhost/uta -c 'select count(*) from uta_20210129b.transcript'
+    # 314227
```
Comment (jsstevenson) [inline docker-compose.yml:24]: is it weird that this number changes?

**Review by jsstevenson — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (korikuzma) [reply docker-compose.yml:24]: I think it's because it's getting newer transcripts that weren't there 3 years ago

Comment (theferrit32) [reply docker-compose.yml:24]: Was `20210129b` built on the same data as `20210129`?

Comment (jsstevenson) [reply docker-compose.yml:24]: Ah, given the versioning I assumed it was trying to stick to the same data. Makes sense!

Comment (korikuzma) [reply docker-compose.yml:24]: Yes, `20210129b` and `20210129` were built on the same data.

Comment (korikuzma) [reply docker-compose.yml:24]: From my local uta:
Screenshot 2023-09-11 at 5 54 43 PM

**Review by theferrit32 — APPROVED** [review]

**Pull request merged**